src/foundry/client/core/issues.d.mts

```diff
@@ -124,21 +24,21 @@ declare global {
 
     /**
      * Add an invalid Document to the module-provided sub-type counts.
-     * @param documentName      - The Document name.
+     * @param cls               - The Document class.
      * @param source            - The Document's source data.
      * @internal
      *
      * @remarks official documentation says "invalid", but the document can be valid, it is simply not yet validated.
      */
     _countDocumentSubType(
-      documentName: DocumentType,
+      cls: Document<any>,
```

Comment:
This should just be `Document.Any`, a helper type we wrote to make sure all the params are tagged correctly.